tools/install_centos_dependencies.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+echo "Installing CentOS 7.6 Dependencies"
+echo "=================================="
+
+# Basic environment
+echo "Checking basic tools..."
+sudo yum install wget git -y
+
+# Protocol Buffers
+echo "Preparing compiling environment..."
+sudo yum install autoconf automake libtool unzip gcc-c++ -y
```

Comment:
I'm wary of the scripts that run ask for my password. Just make the user run `sudo` explicitly when they invoke the script. If you must drop privileges for some operations like network access with git, you can reduce privileges inside the script ($SUDO_USER remains set).